specification/billing/resource-manager/Microsoft.Billing/preview/2018-05-31-preview/examples/BillingAccountsList.json

```diff
@@ -0,0 +1,38 @@
+{
+  "parameters": {
+    "api-version": "2018-05-31-preview",
+    "$filter": "permission eq 'createProject'"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/providers/Microsoft.Billing/billingAccounts/2000100",
+            "name": "2000100",
```

Comment:
What is the master for this name in GTM? If it's going to be the commerce root id, then this would be a GUID generated by MCAPI.

Reply:
Guid should be more accurate. Looks like UX mocks had the wrong examples.